test/models/rhsm_fact_parser_test.rb

```diff
@@ -38,12 +41,22 @@ def test_get_facts_for_interface_with_ip
     end
 
     def test_get_facts_for_interface_without_ip
-      expected_eth1 = {
+      expected_eth2 = {
         'link' => true,
-        'macaddress' => @facts['net.interface.eth1.mac_address'],
+        'macaddress' => @facts['net.interface.eth2.permanent_mac_address'],
         'ipaddress' => nil
       }
-      assert_equal expected_eth1, parser.get_facts_for_interface('eth1')
+      assert_equal expected_eth2, parser.get_facts_for_interface('eth2')
+    end
+
+    def test_get_facts_for_bonded_interface
```

Comment:
This test is the same as `test_get_facts_for_interface_without_ip`, did you mean to make it different?

Reply:
@ianballou Yeah the tests just happened to be the same but with two different purposes.  I rewrote my new test to make its specific purpose more clear.